agent/logger/audit/entry_types.go

```diff
@@ -34,11 +36,11 @@ const (
 	// 4. url
 	// 5. user agent
 	// 6. arn for the entity associated with credentials
-	// 7. event type ('GetCredentials')
+	// 7. event type ('GetCredentials, GetCredentialsExecutionRole')
```

Comment:
In line number 32, it still says version "1". Can you add changes for version "2" under a new comments section? After line 42 perhaps? Similar to how we document state file revision changes?

Reply:
added.